package.json

```
@@ -0,0 +1,61 @@
+{
+	"name": "cli",
+	"version": "0.0.0",
+	"license": "MIT",
+	"bin": {
+		"asyncapi": "dist/cli.js",
+		"@asyncapi/cli": "dist/cli.js"
+	},
+	"engines": {
+		"node": ">=10"
+	},
+	"scripts": {
+		"build": "tsc",
+		"start": "npm run build && node dist/cli.js",
+		"pretest": "npm run build",
+		"pretest:coverage": "npm run build",
+		"test": "jest",
+		"test:coverage": "jest --verbose --coverage"
+	},
+	"files": [
+		"dist/cli.js"
+	],
+	"dependencies": {
+		"@asyncapi/parser": "^1.4.5",
+		"commander": "^7.1.0",
+		"ink": "^3.0.8",
+		"meow": "^9.0.0",
+		"react": "^17.0.1",
+		"reflect-metadata": "^0.1.13",
+		"tsyringe": "^4.5.0"
+	},
+	"devDependencies": {
+		"@sindresorhus/tsconfig": "^0.9.0",
+		"@types/jest": "^26.0.20",
+		"@types/react": "^17.0.2",
+		"chalk": "^4.1.0",
+		"eslint-plugin-react": "^7.22.0",
+		"eslint-plugin-react-hooks": "^4.2.0",
+		"ink-testing-library": "^2.1.0",
+		"jest": "^26.6.3",
+		"ts-jest": "^26.5.3",
+		"typescript": "^4.2.3"
+	},
+	"jest": {
```

enables release automation with semantic-release

```suggestion
"release": {
    "branches": [
      "master"
    ],
    "plugins": [
      [
        "@semantic-release/commit-analyzer",
        {
          "preset": "conventionalcommits"
        }
      ],
      [
        "@semantic-release/release-notes-generator",
        {
          "preset": "conventionalcommits"
        }
      ],
      "@semantic-release/npm",
      "@semantic-release/github"
    ]
  },
    "publishConfig": {
    "access": "public"
  },
	"jest": {
```